app/templates/availability.ejs

```diff
@@ -18,15 +18,15 @@
             <p>Pick a day and time that works for you</p>
 
             <div class="element__wrapper">
-                <div id="cronofy-availability-viewer"></div>
+                <div id="cronofy-date-time-picker"></div>
             </div>
 
         </div>
 
-        <script src="https://elements.cronofy.com/js/CronofyElements.v1.32.0.js"></script>
+        <script src="https://elements.cronofy.com/js/CronofyElements.v1.34.8.js"></script>
         <script>
 
-            const availabilityViewerOptions = {
+            const dateTimePickerOptions = {
                 element_token: "<%= element_token %>",
                 data_center: "<%= data_center %>",
                 target_id: "cronofy-availability-viewer",
```

Comment:
Could we remove the `config` options as these are different on the Date Time Picker.